### ISSUE TITLE
Accessibility swarm on the Story Promo component with Media Indicator

### ISSUE THREAD

**Issue by DenisHdz** [BBC-archive/psammead#560]
**Is your feature request related to a problem? Please describe.**
Carry out a two `devs` accessibility swarm for the `Story Promo` with the `MediaIndicator`.
 
**Describe the solution you'd like**
Go through this [checklist](https://bbc-news.github.io/accessibility-news-and-you/accessibility-news-and-testers) and fix any issue we might face.

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**

**Comment by DenisHdz** [comment]
Blocked on ~[#506](https://github.com/bbc/psammead/issues/506)~

**Comment by dr3** [comment]
(edit me)

❓ = Not done
✅ = Done, pass
❌ = Done, fail

❓ ZoomText Magnifier/Reader Latest Version
With Internet Explorer 11 on Windows (XP/Vista/7/8/10)
(Screen Magnifier with Screen Reader capabilities)

❓ Dragon Naturallyspeaking Version 13
With Internet Explorer 11 on Windows (XP/Vista/7/8/10)
(Speech Recognition)

❓ JAWS Version 17
With Internet Explorer 11 on Windows (XP/Vista/7/8/10)
(Screen Reader)

❓ Read&Write Latest Version
With Internet Explorer 11 on Windows (XP/Vista/7/8/10)
(Reading Solution)

❓ VoiceOver iOS (iPhone/iPad) Latest Version
With Safari (Latest Version) on iOS (Latest Version)
(Screen Reader)
Raised https://github.com/bbc/psammead/issues/743

❓ NVDA (Latest Version)
With Firefox (Latest Version) on Windows (XP/Vista/7/8/10)
(Screen Reader)

**Comment by DenisHdz** [comment]
Blocked on ~[#750](https://github.com/bbc/psammead/issues/750)~, ~[#2096](https://github.com/bbc/simorgh/issues/2096)~ and ~[#1931](https://github.com/bbc/simorgh/issues/1931)~

**Comment by DenisHdz** [comment]
❓ = Not done
✅ = Done, pass
❌ = Done, fail
👀 = Review with UX

## Checklist - Manual ##

  - ### Design ❌     
    https://github.com/bbc/psammead/issues/768 The media indicator wrapper should have padding-bottom: 0px & top padding: 4px when below 400px. This is to ensure there is not padding in addition to the promo padding.

  - ### Color contrast  ✅
     <img width="1210" alt="color-contrast" src="https://user-images.githubusercontent.com/46446236/60648239-6ff18580-9e37-11e9-9cbf-be4c8636cc15.png">

  - ### Color blindness ✅
     <img width="976" alt="color-blindness" src="https://user-images.githubusercontent.com/46446236/60648321-a202e780-9e37-11e9-9745-95f228e9dbfe.png">

  - ### High Contrast mode PC 👀 

     The icon is difficult to see if the background is black.

     ![high-contrast-1](https://user-images.githubusercontent.com/46446236/60657878-f6af5e00-9e49-11e9-9c8c-67d73ad4708d.jpg)

    ![hight-contrast-2](https://user-images.githubusercontent.com/46446236/60657882-f7e08b00-9e49-11e9-9aa1-77cae8bf3aa4.jpg)

  - ### Color preferences changed in Firefox 👀   

     The icon is difficult to see if the background is black.

    <img width="867" alt="color-contrast-black" src="https://user-images.githubusercontent.com/46446236/60654884-bbf6f700-9e44-11e9-8dc6-1d88337e873c.png">

     <img width="416" alt="high-contrast-ff" src="https://user-images.githubusercontent.com/46446236/60605523-9967ce00-9db1-11e9-90fa-3934c47f4373.png">
    
  - ### Text resized to 200% ❌

    Resizing the font to 72px makes the icon look very small

     <img width="1183" alt="text-resized" src="https://user-images.githubusercontent.com/46446236/60605779-0bd8ae00-9db2-11e9-94ce-dc83fd32bab3.png">

  - ### Page zoomed to 200% ✅

    <img width="1197" alt="page-zoomed" src="https://user-images.githubusercontent.com/46446236/60605818-1eeb7e00-9db2-11e9-8b68-2dad93f232c1.png">

  - ### Heading order  ✅

    <img width="1025" alt="heading-order" src="https://user-images.githubusercontent.com/46446236/60653501-0dea4d80-9e42-11e9-8447-65b091406b1b.png">

  - ### Icons ✅
    The `MediaIndicator` allows to pass off-screen text, but it is disabled because this information is provided in the Story Promo headline.

  - ### Keyboard navigation: tab order, focus css, trace tab path ✅

  - ### aXe ✅
    <img width="972" alt="axe" src="https://user-images.githubusercontent.com/46446236/60606026-7b4e9d80-9db2-11e9-8802-f5489fa412a2.png">

  - ### No CSS ✅

    <img width="993" alt="no-css" src="https://user-images.githubusercontent.com/46446236/60607228-cd90be00-9db4-11e9-9b60-ff65a10def05.png">

  - ### No Javascript ✅

  - ### bbc-a11y ✅
     <img width="1125" alt="bbc-a11y" src="https://user-images.githubusercontent.com/46446236/60605710-e9df2b80-9db1-11e9-9000-1b9584f70f66.png">



## Checklist - Assistive Technology ##

Acceptance criteria:
```
Given I use a screen reader
When I navigate to a video promo
And I am navigating via headings and links
Then the media label (video) and media duration (1 minute 33 seconds) should be announced as part of the heading/link
And the media label (video) should be announced first
And the media duration (1 minute 33 seconds) should be announced last
```

 - ###  VoiceOver OS (Mac) (Screen Reader) ✅

   > With Safari (Latest Version) on Mac OS (Latest Version)  

   <img width="833" alt="Screenshot 2019-07-04 at 09 29 12" src="https://user-images.githubusercontent.com/46446236/60651669-9d8dfd00-9e3e-11e9-9138-7568e31ca00e.png">
   <img width="822" alt="Screenshot 2019-07-04 at 09 30 25" src="https://user-images.githubusercontent.com/46446236/60651671-9d8dfd00-9e3e-11e9-9410-2363d0fd1c4d.png">

 - ### VoiceOver iOS (iPhone/iPad) (Screen Reader) ✅

    > Latest Version with Safari (Latest Version) on iOS (Latest Version)

 - ### TalkBack  (Screen Reader)  ❌ 
 
   > Latest Version with Chrome (Latest Version) on Android (Latest Version)

   If an audio story has a duration of `1:00` it is announced as `one o'clock` instead of `one`.

- ### ZoomText (Screen Magnifier with Screen Reader capabilities) ✅

  > Latest Version with Internet Explorer 11 on Windows (XP/Vista/7/8/10)

- ### Dragon Naturallyspeaking  (Speech Recognition) ✅

   > Version 13 with Internet Explorer 11 on Windows (XP/Vista/7/8/10)

- ### JAWS  (Screen Reader) ❌

   > Version 17 with Internet Explorer 11 on Windows (XP/Vista/7/8/10) 

   If an audio story has a duration of `1:00` it is announced as `one o'clock` instead of `one`.

   The story headline is announced as `H2` instead of `H3`

- ### Read&Write  (Reading Solution)  ❌

   > Latest Version with Internet Explorer 11 on Windows (XP/Vista/7/8/10)

   If an audio story has a duration of `1:00` it is announced as `one o'clock` instead of `one`.

- ### NVDA  (Screen Reader) ✅

   > Latest Version with Firefox (Latest Version) on Windows (XP/Vista/7/8/10)

**Comment by dr3** [comment]
Raised https://github.com/bbc/simorgh/issues/2309 for the 1 o'clock bug

**Comment by dr3** [comment]
Media indicator size with zoom bug https://github.com/bbc/psammead/issues/784

**Comment by dr3** [comment]
Med ind contrast https://github.com/bbc/psammead/issues/786

**Comment by dr3** [comment]
Closing as all issues distributed into issues above, Great work everyone
